sormas-ui/src/main/java/de/symeda/sormas/ui/utils/CssStyles.java

```diff
@@ -161,6 +161,10 @@ private CssStyles() {
 	public static final String BUTTON_FONT_SIZE_LARGE = "font-size-large";
 	public static final String GEOCODE_BUTTON_HIGHLIGHT = "geocode-button-highlight";
 
+	// Link styles
+	public static final String LINK_BUTTON = "button";
```

Comment:
```suggestion
	public static final String LINK_BUTTON = "link-button";
```

Reply:
Vaadin Links automatically add the prefix "v-link-" to the style it receives (like Vaadin Buttons with the prefix "v-button-").
Adding "link-" here will produce the class "v-link-link-button".

Reply:
Ah sorry, that makes sense - all fine then :)